pkgs/by-name/st/stratisd/package.nix

```
@@ -29,18 +29,18 @@
 
 stdenv.mkDerivation rec {
   pname = "stratisd";
-  version = "3.7.3";
+  version = "3.8.0";
 
   src = fetchFromGitHub {
     owner = "stratis-storage";
     repo = pname;
```

```suggestion
    repo = "stratisd";
```
https://github.com/NixOS/nixpkgs/issues/277994